app/controllers/application_controller.rb

```diff
@@ -60,6 +61,14 @@ def expired_session?
     false
   end
 
+  def set_sentry_context
+    return unless claim&.id.present? && claim&.application_reference.present? && claim.user&.id.present?
```

Comment:
You have claim& in 2 out of the 3 conditions - but surely if you are expecting claim to possible be nil, it can be nil for all 3 ?